### PR TITLE
[HER-29] Create business project

### PR DESCRIPTION
**Pull request by andreea-dumitrascu**
## Description
Create business project

## Related
[HER-29](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?selectedIssue=HER-29)

### CODE REVIEW

**Comment by danidoff** [comment]
Should we rename the branch with the correct name ? ex. [HER-29]Name ?

File: src/Business/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace Business
+{
+    public class Class1
```
Comment (danidoff) [inline src/Business/Class1.cs:5]: Should we delete this class? It`s not used anywhere in the project.

**Comment by andreea-dumitrascu** [comment]
> Should we rename the branch with the correct name ? ex. [HER-29]Name ?

Done

**Review by danidoff — COMMENTED** [review]
Should we add business reference in Presentation project ?

**Review by TBiannca — APPROVED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**